Gruntfile.js

```diff
@@ -15,11 +15,4 @@ module.exports = function (grunt) {
     var bastion = require('./bastion.js');
 
     bastion(grunt);
-
-    grunt.registerTask('ci', [
-        'eslint',
-        'jshint',
-        'htmlhint',
-        'karma:ci'
-    ]);
```

Comment:
This was an override that only Bastion could see -- https://github.com/Katello/bastion/commit/eaf0eb0b7d8f203b003c8556b18dc08a7dcc5d3c and https://github.com/Katello/bastion/pull/53/files#diff-1a1f7ef6ca0e0de38474f61db2b3ead7R14